### DISCUSSION TITLE
Rustdesk on tablet

### DISCUSSION THREAD

**Discussion by kytril87** [rustdesk/rustdesk#8160] [Q&A]
I have rustdesk installed on my tablet and all services enabled. when I connect from PC to control the tablet remotely, I cannot use the PC keyboard or the scroll wheel on the mouse. Does anyone know what this could be? Only the touch keyboard works. Thank you for your help.

**Comment by rustdesk** [comment]
https://x.com/rustdesk/status/1721795544689922168?t=yrpbb54NJr4R7fvNg2TbZw&s=19
